src/benchmark.cc

```diff
@@ -115,7 +115,13 @@ std::string* GetReportLabel() {
 
 // Global variable so that a benchmark can report an error as a human readable
 // string. If error_message is null no error occurred.
-static std::atomic<const char*> error_message = ATOMIC_VAR_INIT(nullptr);
+#if defined(BENCHMARK_WORKAROUND_VS_2013)
```

Comment:
in this case, it's probably worth just keeping the `#if _MSC_VER <= 1800` here (with a comment explaining it). the extra macro doesn't add anything useful.